ci-operator/step-registry/dpu-operator/workflow/tests/dpu-operator-workflow-tests-ref.yaml

```diff
@@ -0,0 +1,18 @@
+ref:
```

Comment:
Add a README.md in step-registry/dpu-operator, I would like the following described:

1) To avoid leaking of internal resources, we create secrets to `vault.ci.openshift.org` under /kv/selfservice/trigger/token from https://docs.ci.openshift.org/docs/how-tos/adding-a-new-secret-to-ci/
2) How to generate "token_dpu_operator_key" in Jenkins (links to how this is done an maintained)
3) Other important parts you should be sharing for how to make sense of jenkins integration.

Reply:
Also where "PULL_NUMBER" is described in documentation would help.

Reply:
Would be happy to create a README.md. However $PULL_NUMBER is something I think I got from reading some code in this repo. Not sure if I have the documentation for this.